packages/neon_framework/lib/src/models/account.dart

```diff
@@ -26,78 +34,63 @@ abstract interface class Credentials {
 }
 
 /// Account data.
-@JsonSerializable()
 @immutable
-class Account implements Credentials, Findable {
-  /// Creates a new account.
-  Account({
-    required this.serverURL,
-    required this.username,
-    this.password,
-    this.userAgent,
-    @visibleForTesting Client? httpClient,
-  })  : humanReadableID = _buildHumanReadableID(username, serverURL),
-        id = sha1.convert(utf8.encode('$username@$serverURL')).toString() {
-    final cookieStore = NeonStorage().cookieStore(
-      accountID: id,
-      serverURL: serverURL,
-    );
+abstract class Account implements Credentials, Findable, Built<Account, AccountBuilder> {
+  factory Account([void Function(AccountBuilder)? updates]) = _$Account;
 
-    client = NextcloudClient(
-      serverURL,
-      loginName: username,
-      password: password,
-      appPassword: password,
-      userAgent: userAgent,
-      cookieJar: cookieStore != null ? CookieJarAdapter(cookieStore) : null,
-      httpClient: httpClient,
-    );
```

Comment:
With these changes the client is no longer cached (and all the other fields as well). Can we somehow keep them as fields, but completely ignore them in built_value?

Reply:
I guess one possible way would be using Expando to cache it outside of the class, but maybe this isn't necessary after all since I don't see something that is super expensive about this behavior.

Reply:
After thinking more about this we should definitely do this. Otherwise all the Expandos used inside the client itself are useless and execute a lot of unnecessary memory operations.
@Leptopoda let me know if there is any cleaner way than using Expando, otherwise I will just go with it.

Reply:
I can't really follow.
Can you explain it in another way?

Reply:
Oh I didn't know about the memoized annotation, but this solves the caching problem I was going to fix by using Expando.